scripts/sonar-analysis.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
The same comments are for promote.sh
- consider moving to .cirrus/
- some sonar properties should be set on the project, not on the call:
  - sonar.projectName, sonar.projectKey, sonar.organization -> this is read from package.json
  - sonar.sources, sonar.tests, sonar.exclusions, sonar.javascript.lcov.reportPath, sonar.coverage.exclusions -> sonar-project.properties